packages/back-end/src/services/experiments.ts

```diff
@@ -612,7 +647,10 @@ export async function createManualSnapshot({
     experiment,
     phaseIndex,
     orgPriorSettings: orgPriorSettings,
-    settings: analysisSettings,
+    orgDisabledPrecomputedDimensions: false,
```

Comment:
question: Should we get this from org settings?

Reply:
No, this is manual snapshots that come from users manually enter data and are basically (if not actually) deprecated. Precomputed dimensions doesn't make sense for these in any case.